templates/admin/org/list.tmpl

```diff
@@ -13,15 +13,16 @@
 			{{template "admin/base/search" .}}
 		</div>
 		<div class="ui attached table segment">
+			<script src="{{AppSubUrl}}/vendor/plugins/sortt/sortt.js" charset="utf-8"></script>
 			<table class="ui very basic striped table">
 				<thead>
 					<tr>
-						<th>ID</th>
-						<th>{{.i18n.Tr "admin.orgs.name"}}</th>
+						<th onclick="sortt('oldest','newest')">ID</th>
```

Comment:
imho it would be better if these values would be data attributes and onclick events would be bind using jquery

Reply:
ok I'll change it ...

Reply:
[sortt.js v1.2](https://gitea.com/6543/gitea_sortt/releases) ...

Reply:
done at https://github.com/go-gitea/gitea/pull/7980/commits/c7237b6dc675297a2f069db3ca5bc8588dbf8459